src/routes.js

```diff
@@ -21,6 +21,16 @@ const middlewareRoute = async router => {
     transformerMiddleware(),
     mapBodyMiddleware()
   )
+
+  router.get(
+    MIDDLEWARE_PATH_REGEX,
+    initiateContextMiddleware(),
+    parseBodyMiddleware(),
+    requestsMiddleware(),
+    validateBodyMiddleware(),
+    transformerMiddleware(),
+    mapBodyMiddleware()
+  )
```

Comment:
Instead of duplicating the router for each method, lets maybe run through the array of allowed methods and build them up specifically. Something like

```
forEach(allowed_methods, (method) -> {
  router[method](
    MIDDLEWARE_PATH_REGEX,
    initiateContextMiddleware(),
    parseBodyMiddleware(),
    requestsMiddleware(),
    validateBodyMiddleware(),
    transformerMiddleware(),
    mapBodyMiddleware()
  )
})
```

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/79/commits/f984b7a77a6c6e4024d66aab4a08a28dc05161e7